common/djangoapps/third_party_auth/management/commands/saml.py

```diff
@@ -166,7 +173,7 @@ def _check_provider_configurations(self):
             'outdated_count': {'count': outdated_count, 'requires_attention': True},
             'site_mismatch_count': {'count': site_mismatch_count, 'requires_attention': True},
             'slug_mismatch_count': {'count': slug_mismatch_count, 'requires_attention': True},
```

Comment:
1. Since this is now an `INFO`, should it no longer require attention? 
2. Should the count output move to the `"CHECK SUMMARY:"` section as well? 
```suggestion
            'slug_mismatch_count': {'count': slug_mismatch_count, 'requires_attention': False},
```

Reply:
since that is treated as information so i have made the required changes and moved the count output in check_summary.